### DISCUSSION TITLE
Alarm flipping between 0 and 0.0

### DISCUSSION THREAD

**Discussion by humbeecc** [yozik04/nibe#97] [General]
Since more than a week, the logbook shows almost every second a "F1145 Alarm changed to 0.0" and "F1145 Alarm changed to 0" message.
The F1145 is connected via the esphome-nibe component.
Maybe some int or float typing issue was introduced lately?

**Comment by yozik04** [comment]
Logbook where?

**Comment by elupus** [comment]
I guess home assistant. Could we be parsing the value different depending of its from the RMU data and polled?

**Reply by yozik04** [reply]
Totally possible, need to check.

**Reply by yozik04** [reply]
But did you released a new HA Integration version with new lib version?

**Reply by elupus** [reply]
We have 2.0.0. So new but not latest.

**Comment by humbeecc** [comment]
Indeed home assistant.
Maybe important: the alarm status is also part of a limited set that is automatically sent every x seconds. I had to upload some file via usb to configure this on the F1145. Don't remember the exact naming for this procedure.

**Comment by yozik04** [comment]
I think I have a fix. Need to validate and release.

**Comment by yozik04** [comment]
Fixed in 2.1.2.

**Comment by humbeecc** [comment]
Thank you. It is indeed solved in the Home Assistant 2023.3.5 release.
